### PR TITLE
Code review-trial-project

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
updated style.css

### CODE REVIEW

**Review by kinginthenorthcodez — COMMENTED** [review]
nice use of descriptive CSS selectors 👍🏽 
Good PR 👍🏽 
Descriptive README 👍🏽 
Good clear Commits 👍🏽 
Semantics HTML practice 👍🏽 
APPROVED ♻️ 
There is nothing else to say other than its time to merge! 🏁

**Review by OLIPLICHE — COMMENTED** [review]
Hi @kinginthenorthcodez, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
- You have submitted a project, which does not have at least 2 comments in the review.

- Kindly read the project requirements. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/style.css [hunk 1]
```diff
@@ -30,6 +30,7 @@
   padding-top: 24px;
   text-decoration-line: underline;
   color: #fff;
+  font-variant: small-caps;
```
Comment (kinginthenorthcodez) [inline css/style.css:33]: I would suggest styling the links without an underline here 👍🏽 because it would mess up the look and feel of your design.

Comment (kinginthenorthcodez) [inline css/style.css:33]: It's okay to try to use font-variant ,but let's stick to the original design of topography 👍🏽

File: css/style.css [hunk 2]
```diff
@@ -30,6 +30,7 @@
...
 }
 
 .social-links a:hover {
```
Comment (kinginthenorthcodez) [inline css/style.css:36]: check and make sure your indentation is 2 spaces 👍🏽

**Review by kinginthenorthcodez — COMMENTED** [review]
Almost there, Please make a few changes according to the comments 💯 
happy coding 😃

**Pull request merged**